### PR TITLE
TGMC Black suit

### PR DESCRIPTION
**Pull request by David-Stormwell**
## About The Pull Request

Front
Back
Right
Left

## Why It's Good For The Game

Snazzy, and for those folks who can't dosh up the money in real life for a suit! :D

## Changelog

:cl:
add: A Marine full-piece suit has been added to the dress tab of the clothing vendor! Sprites courtesy of _a Ghost_
\:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
image

**Comment by David-Stormwell** [comment]
For reasons, I will be making this fit into the suit slot.

**Comment by MLGTASTICa** [comment]
wheres the mandatory +500 hard melle armor?

**Comment by FIXTHERADBOXPLEASE** [comment]
Here come the Men in Black...

**Comment by Kuro020** [comment]
how about you take a pic without the satchel so I can actually see the sprite 4head

**Comment by Aporhtonoma** [comment]
image

**Comment by David-Stormwell** [comment]
> how about you take a pic without the satchel so I can actually see the sprite 4head

how about I fucin kill u!!!

**Comment by Coastfront** [comment]
If this doesen't get merged then the 3rd coming of christ will never happen

**Comment by FosterGrime-Heymynameslouie** [comment]
old dress uniform was pretty good

File: code/modules/clothing/under/marine_uniform.dm [hunk 1]
```diff
@@ -113,7 +113,12 @@
 	icon_state = "marine_striped"
 	item_state = "marine_striped"
 	adjustment_variants = list()
-
+/obj/item/clothing/under/marine/black_suit
+	name = "\improper marine black suit"
+	desc = "A easy fitting black suit, somehow exactly your size."
+	icon_state = "marine_suit"
+	item_state = "marine_suit"
+	adjustment_variants = list()
```
Comment (TiviPlus) [inline code/modules/clothing/under/marine_uniform.dm:121]:
```suggestion

/obj/item/clothing/under/marine/black_suit
	name = "\improper marine black suit"
	desc = "A easy fitting black suit, somehow exactly your size."
	icon_state = "marine_suit"
	item_state = "marine_suit"
	adjustment_variants = list()

```

**Review by TiviPlus — APPROVED** [review]
https://youtu.be/zyNyHark4xk

**Pull request merged**